examples/streaming/timesales.py

```diff
@@ -38,17 +66,26 @@ def initialize(self):
         """
         self.tda_client = easy_client(
             api_key=self.api_key,
-            redirect_uri='https://localhost:8080',
-            token_path=self.credentials_path)
-        self.stream_client = StreamClient(self.tda_client, account_id=self.account_id)
+            redirect_uri="https://localhost:8080",
+            token_path=self.credentials_path,
```

Comment:
Is that an errant extra comma at the end on line 70?
`token_path=self.credentials_path,`

Reply:
I just now learned that trailing commas are allowed in python. Ignore my comment.